modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCSSOConsentTestCase.java

```diff
@@ -0,0 +1,343 @@
+        serviceProvider = createApplication(new ServiceProvider(), playgroundApp);
+        client = HttpClientBuilder.create().setDefaultCookieStore(cookieStore).build();
+    }
+
+    @AfterClass(alwaysRun = true)
+    public void testClear() throws Exception {
+
+        deleteUser(user);
+        deleteApplication(playgroundApp);
+        clear();
+    }
+
+    @Test(groups = "wso2.is", description = "Test consent management after updating " +
+            "the application claim configurations")
+    public void testConsentWithAppClaimConfigUpdate() throws Exception {
+
+        testSendAuthenticationRequest(playgroundApp, client);
+        testAuthentication(playgroundApp);
+        Assert.assertEquals(claimsToGetConsent, "0_Email,1_First Name",
+                "Incorrect claims to request the consent");
+        testConsentApproval(playgroundApp);
+        testGetAccessToken(playgroundApp);
+        testUserClaims();
+
+        performOIDCLogout();
+        updateApplication(playgroundApp, serviceProvider);
+
+        // Login again with updated claim configurations.
+        testSendAuthenticationRequest(playgroundApp, client);
+        testAuthentication(playgroundApp);
+        Assert.assertEquals(claimsToGetConsent, "0_Last Name",
+                "Incorrect claims to request the consent");
+    }
+
+    public void testSendAuthenticationRequest(OIDCApplication application, HttpClient client)
+            throws Exception {
+
+        List<NameValuePair> urlParameters = OIDCUtilTest.getNameValuePairs(application);
+        HttpResponse response = sendPostRequestWithParameters(client, urlParameters, String.format
+                (OIDCUtilTest.targetApplicationUrl, application.getApplicationContext() + OAuth2Constant.PlaygroundAppPaths
+                        .appUserAuthorizePath));
+        Assert.assertNotNull(response, "Authorization request failed for " + application.getApplicationName() + ". "
+                + "Authorized response is null");
+
+        Header locationHeader = response.getFirstHeader(OAuth2Constant.HTTP_RESPONSE_HEADER_LOCATION);
+
+        Assert.assertNotNull(locationHeader, "Authorization request failed for " + application.getApplicationName() +
+                ". Authorized response header is null");
+        EntityUtils.consume(response.getEntity());
+
+        response = sendGetRequest(client, locationHeader.getValue());
+        Assert.assertNotNull(response, "Authorization request failed for " + application.getApplicationName() + ". "
+                + "Authorized user response is null.");
+
+        Map<String, Integer> keyPositionMap = new HashMap<>(1);
+        keyPositionMap.put("name=\"sessionDataKey\"", 1);
+        List<DataExtractUtil.KeyValue> keyValues = DataExtractUtil.extractDataFromResponse(response,
+                keyPositionMap);
+        Assert.assertNotNull(keyValues, "sessionDataKey key value is null for " + application.getApplicationName());
```

Comment:
Let's have the sentence case followed for the error message.

Reply:
Fixed https://github.com/wso2/product-is/pull/11407/commits/4fe18ee0052418875fffa6a22be116143ea73ebb